### DISCUSSION TITLE
Your project? 🚀

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#819] [General]
What are you building, and trying to achieve on top of pH7Builder? What's your project and your goal?

**Comment by WolfDieterKattnig** [comment]
Hi, I am trying to make a free-of-charge dating/matchmaking Site in Styria/Austria.
At the moment, I am just testing, but would like to go on-line by the beginning of summer.

**Comment by Baillie11** [comment]
I am building a Adult Dating site in Australia for Australians.

I literally launched today. Doing a soft launch now providing site for free for the first 100 sign ups.

www.yolodationg.com.au

**Reply by hobb7t** [reply]
wasn't that much successful?

**Comment by dreamlove66** [comment]
Bonjour , j ai crée un site de rencontre libertine mais j ai un big la première page ne veut plus s afficher https://www.community-libertine.fr/
les autre page fonctionne https://www.community-libertine.fr/super-profil/74  si quelqu'un peut m'aider merci d avance

**Comment by AndrewHalifax** [comment]
Im working to building a  LGBTQ2+ and Non Exclusive dating site. https://dudedating.com 
Ideally trying to make it as open and available to everyone as POF would be but less restrictive with viewing a potential match.
